ci/terraform/audit-processors/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "deployer_role_arn" {
+  default     = ""
+  description = "The name of the AWS role to assume, leave blank when running locally"
+  type        = string
+}
+
+variable "environment" {
+  type = string
+}
+
+variable "aws_region" {
+  default = "eu-west-2"
```

Comment:
I don't have much experience with terraform so it might be fine, but does the type need to be listed for this variable?